src/webui/webapplication.cpp

```diff
@@ -699,6 +715,28 @@ void WebApplication::sessionInitialize()
         sessionStart();
 }
 
+void WebApplication::apiKeySessionInitialize()
+{
+    Q_ASSERT(!m_currentSession);
+
+    QString sessionId;
+    if (const QString apiKey = parseAuthorizationHeader(m_request.headers.value(u"authorization"_s));
```

Comment:
IIRC, "Authorization" header is not so appropriate for API key. I would use "X-API-Key" instead.

Reply:
As someone who worked in the API key management space for over 5 years, I assure you the `Authorization` header is the **only** appropriate place for an api key. Custom headers should not be used for this.

Reply:
While not actually a bearer token, it's become a bit of a standard on the internet to use `Bearer` for static api keys. It really is just what everyone does.